src/Menu/Address.java

```diff
@@ -0,0 +1,34 @@
+package Menu;
+
+public class Address {
+
+        private String roadName;
+        private Integer houseNum;
```

Comment:
```suggestion
        private Integer number;
```

como vc vai lidar com apartamentos, que tem blocos e etc?